### DISCUSSION TITLE
Force Recovery mode in A203 Mini PC!

### DISCUSSION THREAD

**Discussion by AhmedFawzy0** [Seeed-Studio/wiki-documents#128] [Q&A] [answered]
There is no recovery key, how can I force recovery?
Just power and reset doesn't work

<img width="806" alt="image" src="https://user-images.githubusercontent.com/89322519/230556373-4be91e8b-7004-4110-a562-722d0548bade.png">

**Comment by MatthewJeffson** [comment, marked as answer]
Hi, on the Step 4, the "Recovery Key" really should be "Reset Key". I have changed that. Thank you for pointing it out!
